MiniSim/Views/Preferences.swift

```diff
@@ -11,8 +11,26 @@ import KeyboardShortcuts
 import LaunchAtLogin
 
 struct Preferences: View {
+    var menuImages = [ "menu_icon_1", "menu_icon_2", "menu_icon_3" ]
+    @State var menuImageSelected = 0
```

Comment:
I think you can use the `@AppStorage` property wrapper to save directly to user defaults 👍🏻

Reply:
From my tests, AppStorage can be used, but the Picker's appearance won't update without State. I can't add both properties in one variable.

Reply:
I've used `AppStorage` for Picker in this file: `MiniSim/Views/Onboarding/SetupView.swift` - check it out